update vale and fix ini (#56)
ConsenSys/doc.goquorum: docs/HowTo/GetStarted/Cakeshop.md

    ```

    !!! warning

>         This command will unpack the war file and set up the Cakeshop data folders but will not actually start a node

Reviewer: ```suggestion
        This command unpacks the war file and sets up the Cakeshop data folders but does not actually start a node
```
